test/test_deltafilewrapper.cpp

```diff
@@ -970,7 +976,7 @@ TEST_CASE( "Delta File Wrapper" )
             },
             "old": {
               "attributes": {
-                "attachment": null,
+                "attachment": %3,
```

Comment:
Hm, who causes `""` instead of `null`? I think `null` is more explicit that the user did not set any value and `""` is like - they did, but we such at obtaining the file path.